src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1186,6 +1186,24 @@ internal virtual void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleB
 
         internal abstract bool IsNullableAnalysisEnabled();
 
+        /// <summary>
+        /// Gets the resolution priority of this method, 0 if not set.
+        /// </summary>
+        /// <remarks>
+        /// Do not call this method from early attribute binding, cycles will occur.
+        /// </remarks>
+        internal int OverloadResolutionPriority => CanHaveOverloadResolutionPriority ? (TryGetOverloadResolutionPriority() ?? 0) : 0;
+
+        internal abstract int? TryGetOverloadResolutionPriority();
+
+        internal bool CanHaveOverloadResolutionPriority =>
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

nit: accessibility likely can be further tightened to protected

Reply:
I'm using it in the extension method assertion. I'd remove it, but it actually did catch an issue with how I'd written the body.